### ISSUE TITLE
error GPRmax

### ISSUE THREAD

**Issue by Harvsie82** [gprMax/gprMax#39]
Hi...

I have installed gprMax several times in several windows computers, even with help of this video https://www.youtube.com/watch?v=dvC0_7qFVbE, but no matter what I do, alway I get this error

![image](https://cloud.githubusercontent.com/assets/17809927/13787360/a759dbc0-eaa9-11e5-863c-55b6a6d8f677.png)

Please help me, I don´t know what is wrong

**Comment by craig-warren** [comment]
@Harvsie82 can you post a listing of the directory ``C:\Users\Harvey\Desktop\gprMax-v.3.0.0.b19\gprMax`` please.

**Comment by Harvsie82** [comment]
Thank you for your help... This is the listing of the directory

![image](https://cloud.githubusercontent.com/assets/17809927/13824345/a10a8aa8-eb7b-11e5-956b-8494df716e0c.png)

**Comment by craig-warren** [comment]
@Harvsie82 just checked the release you are trying to use (v.3.0.0b19) on my Windows 7 64-bit virtual machine and it worked OK. You seem to have the modules copied to the correct location.

Can you tell me what version of Windows you are using, and confirm your machine is 64-bit?

**Comment by Harvsie82** [comment]
Yes, I have installed v.3.0.0b21, 3.0.0b20, v3.0.0.b1 and did't work, this is a print screen of my windows version

![image](https://cloud.githubusercontent.com/assets/17809927/13827902/e65ebd5c-eb8c-11e5-90e8-544e20d9161c.png)

But, I´ve installed in other computer v.3.0.0b20 and I got this error too...

![image](https://cloud.githubusercontent.com/assets/17809927/13828296/ccd10ae6-eb8e-11e5-8933-3966858ec073.png)

is a windows 10

![image](https://cloud.githubusercontent.com/assets/17809927/13828397/55029420-eb8f-11e5-8a8d-ed0def5c3686.png)

**Comment by craig-warren** [comment]
@Harvsie82 OK, let's stick with the Windows 7 64-bit machine and v.3.0.0b19 for now. It looks like you copied the modules to the correct location, and your Miniconda install is OK. So, just to be sure can you:

* Open a new Command Window
* cd to the top level gprMax-v.3.0.0b19 directory
* Activate the gprMax Miniconda environment
* Run ``python -m gprMax user_models\cylinder_Ascan_2D.in``

**Comment by Harvsie82** [comment]
Ok...

![image](https://cloud.githubusercontent.com/assets/17809927/13831936/70beb52a-eba5-11e5-9036-46d4bdda69f2.png)

**Comment by craig-warren** [comment]
No, run the command from the top level gprMax-v3.0.0b19 directory, i.e. up one directory from where you are.

**Comment by Harvsie82** [comment]
Oh, I´m sorry for that... you had said before.  Nevertheless...
![image](https://cloud.githubusercontent.com/assets/17809927/13852138/d8de4268-ec2e-11e5-8c77-5cd0bd14ff75.png)

**Comment by craig-warren** [comment]
@Harvsie82 one further thing to check is that you completed step 3 of the installation and in your case ran ``vc_redist.x64.exe``?  i.e.

* Install the Microsoft Visual Studio 2015 C++ Redistributable (vc_redist.x86.exe for 32-bit or vc_redist.x64.exe for 64-bit) from https://www.microsoft.com/en-us/download/details.aspx?id=48145.

**Comment by Harvsie82** [comment]
Ok...  everything is good...  a lots of thank... I really apreciate your help Craig...

**Comment by craig-warren** [comment]
No problem...but can you let us know what fixed it. It is useful to know if we need to improve documentation etc...

**Comment by Harvsie82** [comment]
I think the problem was Windows 10, because it doesn't work in 3 computers when I installed before.  And in windows 7 computer when finally works, I followed guidance step by step, but when I found the error I finish the procedure and I didn't  install the Microsoft Visual Studio.   Nevertheless I'm going to install GPRmax in windows 10 again and I will tell you whatever I'll find about this.  Again...  Thank you Craig
